### ISSUE TITLE
Can't upgrade 0.9.13 --> 1.0.0 with PostgreSQL backend

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-server#65]
```
14:36:29.386 [Main] INFO  org.flywaydb.core.internal.command.DbValidate - Successfully validated 25 migrations (execution time 00:00.013s)
14:36:29.392 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Current version of schema "LINSTOR": 2019.07.09.10.42
14:36:29.392 [Main] WARN  org.flywaydb.core.internal.command.DbMigrate - outOfOrder mode is active. Migration of schema "LINSTOR" may not be reproducible.
14:36:29.393 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2019.07.23.13.37 - Add resource- and volume-groups
Error: 
CREATE TABLE RESOURCE_GROUPS
(
UUID CHARACTER(36) NOT NULL,
RESOURCE_GROUP_NAME VARCHAR(48) NOT NULL, -- PK
RESOURCE_GROUP_DSP_NAME VARCHAR(48) NOT NULL,
DESCRIPTION VARCHAR(1024) NULL, -- nullable
LAYER_STACK VARCHAR(1024) NULL, -- nullable, json list of strings
REPLICA_COUNT INT NULL, -- nullable, if null, this rscGrp is not usable with auto-place
POOL_NAME VARCHAR(48) NULL, -- nullable, no foreign key, to allow referencing not yet or no longer existing storage pools
DO_NOT_PLACE_WITH_RSC_REGEX VARCHAR(4096) NULL, -- nullable
DO_NOT_PLACE_WITH_RSC_LIST VARCHAR(4096) NULL, -- nullable, json list of strings
REPLICAS_ON_SAME BLOB NULL, -- nullable, json list of strings
REPLICAS_ON_DIFFERENT BLOB NULL, -- nullable, json list of strings
ALLOWED_PROVIDER_LIST VARCHAR(1024) NULL, -- nullable, json list of strings
DISKLESS_ON_REMAINING BOOLEAN NULL, -- nullable
CONSTRAINT PK_RG PRIMARY KEY (RESOURCE_GROUP_NAME),
CONSTRAINT UNQ_RG_UUID UNIQUE (UUID),
CONSTRAINT CHK_RG_NAME CHECK (UPPER(RESOURCE_GROUP_NAME) = RESOURCE_GROUP_NAME AND LENGTH(RESOURCE_GROUP_NAME) >= 2),
CONSTRAINT CHK_RG_DSP_NAME CHECK (UPPER(RESOURCE_GROUP_DSP_NAME) = RESOURCE_GROUP_NAME)
)
14:36:29.409 [Main] ERROR org.flywaydb.core.internal.command.DbMigrate - Migration of schema "LINSTOR" to version 2019.07.23.13.37 - Add resource- and volume-groups failed! Changes successfully rolled back.
14:36:29.422 [Main] ERROR LINSTOR/Controller - SQL State  : 42704
Error Code : 0
Message    : ERROR: type "blob" does not exist
  Position: 634
 [Report number 5D4C336C-00000-000000]
```

[ErrorReport-5D4C336C-00000-000000.log](https://github.com/LINBIT/linstor-server/files/3482270/ErrorReport-5D4C336C-00000-000000.log)

**Comment by rp-** [comment]
Bummer, we just found out our postgresql and mariadb tests, didn't actually test what we expected and so this slipped in.
We will probably make a 1.0.1 release tomorrow fixing this.

**Comment by rp-** [comment]
fixed in 1.0.1
